### PR TITLE
Declare editing state on new message modal

### PR DESCRIPTION
**Pull request by GretaD**
ref https://github.com/nextcloud/mail/pull/6172#issuecomment-1638668073

### CODE REVIEW

**Review by ChristophWurst — DISMISSED** [review]
States were removed. Remove/fix the usage in the template

**Comment by GretaD** [comment]
> States were removed. Remove/fix the usage in the template

I was wondering which one was the case, removing or adding the leftovers. I added the leftovers because you added the state it on this PR: https://github.com/nextcloud/mail/pull/7691/files after removing it from Composer component. Just FYI why i kept it. But will remove it again and make it work without it then

File: src/components/NewMessageModal.vue [hunk 1]
```diff
@@ -118,6 +118,9 @@ import { mapGetters } from 'vuex'
 import MinimizeIcon from 'vue-material-design-icons/Minus.vue'
 import { deleteDraft, saveDraft, updateDraft } from '../service/DraftService'
 
+const STATES = Object.seal({
+	EDITING: 0,
+})
```
Comment (st3iny) [inline src/components/NewMessageModal.vue:123]:
```suggestion
```

Revert

File: src/components/NewMessageModal.vue [hunk 2]
```diff
@@ -146,6 +149,8 @@ export default {
 			modalFirstOpen: true,
 			cookedComposerData: undefined,
 			changed: false,
+			state: STATES.EDITING,
+			STATES,
```
Comment (st3iny) [inline src/components/NewMessageModal.vue:153]:
```suggestion
```

Revert

**Review by st3iny — COMMENTED** [review]
Tested and works. Please revert unrelated changes (around states).

**Review by st3iny — APPROVED** [review]

**Pull request merged**